build.override.targets

```diff
@@ -9,6 +9,13 @@
     </TraversalBuildDependsOn>
   </PropertyGroup>
   
+    <PropertyGroup>
+    <TraversalCleanDependsOn>
+      WcfClean;
```

Comment:
We do need 2 different targets, because the conditions under which they execute are different (one is triggered only if WcfSetup discovered it started the service, the other is called unconditionally on a Clean).  But I'll make only one of them do the work, and make the other one depend on the worker one when the right conditions are met.